def raise_exception_msg(msg="Custom Name Error Message"):
    raise NameError(msg)